genswagger/genswagger.go

```diff
@@ -0,0 +1,284 @@
+package genswagger
```

Comment:
The `genswagger` package should have a package level comment that at least contains the same information as is found in the README.md for the package.